### DISCUSSION TITLE
blog/rails-debugger-rdbg-vscode

### DISCUSSION THREAD

**Discussion by giscus** [harrison-broadbent/railsnotes-comments#10] [RailsNotes Comments]
# blog/rails-debugger-rdbg-vscode

A short introduction to the rdbg/debugger extension for VS Code, plus a guide on how to set it up correctly to debug your Ruby on Rails applications, as well as your RSpec specs. The VS Code extension takes 5 minutes to setup and it's great!

https://railsnotes.xyz/blog/rails-debugger-rdbg-vscode

**Comment by Samuelodan** [comment]
Hi, thanks for writing this article.
I was trying out Overmind to see if I would be happy to replace Foreman with it, but I couldn't use the debugger with Overmind because it was waiting for the connection. I didn't wanna have to run `rdbg --attach` from another window like I would with Foreman, so I sought an explanation. And that's how I ended up here.

TIL the purpose of `RUBY_DEBUG_OPEN=true` and how it was the culprit.


Thanks for your [Overmind article](https://railsnotes.xyz/blog/overmind-better-bin-dev-for-your-procfile-dev#connect-to-a-running-process-with-overmind-connect) too. I loved the intro.
